frame/instrumental/src/lib.rs

```diff
@@ -559,7 +559,6 @@ pub mod pallet {
 					<T::Vault as StrategicVault>::withdraw(&vault_id, issuer, amount),
 				FundsAvailability::MustLiquidate =>
 					<T::Vault as StrategicVault>::withdraw(&vault_id, issuer, amount),
-				FundsAvailability::None => {},
```

Comment:
I think you should capture the case `FundsAvailability::None` too and don't throw an error. What do you think? (@belousm, @kevin-composable)